src/actions/bookingDaysOffActions.js

```diff
@@ -60,119 +43,158 @@ const validateBody = (body) => {
             rejectStatus += "End date is invalid\n";
         }
         rejectStatus += isDateIntervalOk(body.endDate, "End date");
-        if (rejectStatus !== "") reject({
-            "status": 400,
-            "message": rejectStatus
-        })
-        else
-
-        resolve();
-    });
-}
-
+        if (rejectStatus !== "")
+        reject({
+            status: 400,
+            message: rejectStatus
+        });
+    else resolve();
+});
+};
 
-const validateUserExists = (body, users) => {
+const checkUserExistsInDB = userId => {
```

Comment:
This method might get duplicated, inspect other branches

Reply:
tot noi folosim functia aceasta doar ca intr-un branch vrem sa trimitem mai departe userul in resolve iar in celelalte nu. Sa o folosim pe aceeasi care trimite userul si in celelalte cazuri si sa ignore urmatoarea functie din chain-ul de promises  parametrul asta? e ok asa?

Reply:
Sounds good!
Make sure the name also reflects what it does 👍